src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+.main {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.content__form {
+  margin: 20px 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search__bar {
+  display: flex;
+  height: 70px;
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)